ark/analysis/dimensionality_reduction.py

```diff
@@ -78,15 +74,15 @@ def visualize_dimensionality_reduction(cell_data, columns, category, color_map="
         color_map (str):
             Name of MatPlotLib ColorMap used, default is Spectral
         algorithm (str):
-            Name of dimensionality reduction algorithm, default is UMAP
+            Name of dimensionality reduction algorithm, must be UMAP, PCA, or tSNE, default is UMAP
```

Comment:
If the default is present in as a default in the definition, I don't think we need to redefine it in the docstring. If the default isn't immediately obvious (i.e., it results from None being assigned something), then it's worth having in the docstring

Reply:
I think we can remove "default is spectral" and "default is UMAP" from docstring, since they're both defined at the top